### ISSUE TITLE
Show example images of contrasts that are to be selected by a few functions

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#3062]
Functions such as `sct_deepseg_sc`, `sct_register_to_template`, etc. require an argument "-c" which corresponds to the image contrast. It might not be obvious for users how the image should look like, for a given contrast. 

For this reason, it would be advisable to add examples of contrasts in the documentation and to refer to it in the script usages.

Example:

t1
![Screen Shot 2020-11-25 at 1 25 03 PM](https://user-images.githubusercontent.com/2482071/100267605-b3afc800-2f21-11eb-9763-b3ed75b2e33c.png)

t2
![Screen Shot 2020-11-25 at 1 24 12 PM](https://user-images.githubusercontent.com/2482071/100267618-b6aab880-2f21-11eb-9ac7-473d2bfebd83.png)

t2s
![Screen Shot 2020-11-25 at 1 24 10 PM](https://user-images.githubusercontent.com/2482071/100267622-b90d1280-2f21-11eb-9075-f3bd3ed6b54f.png)

dwi
![Screen Shot 2020-11-25 at 1 24 30 PM](https://user-images.githubusercontent.com/2482071/100267626-bc080300-2f21-11eb-8af8-4424fbaa2417.png)

**Comment by joshuacwnewton** [comment]
Something similar to the note in the [Segmentation tutorial](https://spinalcordtoolbox.com/en/stable/user_section/tutorials/spinalcord_segmentation.html), then?

![Screenshot from 2020-11-25 13-33-27](https://user-images.githubusercontent.com/16181459/100268346-ddb5ba00-2f22-11eb-8a32-f7bed0d314e9.png)

[Substitutions](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions) might be good for this.

> reST supports “substitutions” (ref), which are pieces of text and/or markup referred to in the text by |name|.

> If you want to use some substitutions for all documents, put them into rst_prolog or rst_epilog

So, we could define, say, `|contrast_note|` and then use that wherever we want to include the note.

**Comment by jcohenadad** [comment]
ah! it's already there! cool. Then in that case we just need to redirect to this in the functions' usage.  I was just talking with a user who had not read the doc online, but who had read the usage.

**Comment by joshuacwnewton** [comment]
Good idea. There should definitely be a link between the basic usage and the in-depth tutorial.

Mind you, that's just for spinal cord segmentation. As you rightfully pointed out, the `-c` option is relevant for other SCT commands too (e.g. `sct_register_to_template`). 

I'm working on the registration to template tutorial right now, so I'll definitely keep this in mind as I'm writing. I'll try to ensure that the docs are structured in a way that makes it easy for folks to arrive at related sections.
